lib/rules/disallow-keywords-in-comments.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Disallows keywords in your comments, such as TODO or FIXME
+ *
+ * Type: `Boolean` or `String` or `Array`
+ *
+ * Values:
+ * - `true`
+ * - `'\b(word1|word2)\b'`
+ * - `['word1', 'word2']`
+ *
+ * #### Examples
+ *
+ * ```js
+ * "disallowKeywordsInComments": true
+ * "disallowKeywordsInComments": '\b(word1|word2)\b'
+ * "disallowKeywordsInComments": ['word1', 'word2']
+ * ```
+ *
+ * #### Invalid:
+ * ```
+ * // ToDo
+ * //TODO
+ * /** fixme *\/
+ * /**
+ *  * FIXME
+ *  *\/
+ * ```
+ */
+
+var assert = require('assert');
+
+// default keywords
+var keywords = ['todo', 'fixme'];
+
+var keywordRegEx;
+var buildKeywordRegEx = function(suppliedPattern) {
+    var pattern;
+
+    if (!suppliedPattern) {
+        pattern = '\\b(';
+        pattern += keywords.join('|');
+        pattern += ')\\b';
```

Comment:
Why not just munge all together on one line? 

I'll take care of that post-merge.